copulas/bivariate/base.py

```diff
@@ -54,7 +54,7 @@ class Bivariate(object):
     theta_interval = []
     invalid_thetas = []
     theta = None
-    tau = None
+    tau = None  # TODO: Why is this an attribute?
```

Comment:
Tau isn't a parameter. It's estimated once from the data and never used again. Why is it an attribute?